src/firebase/config.js

```diff
@@ -12,10 +12,10 @@ const firebaseConfig = {
     measurementId: "G-KE55SGFC5L"
 };
 
-firebase.initializeApp(firebaseConfig);
+initializeApp(firebaseConfig);
 
-const projectFirestore = firebase.firestore();
-const timestamp = firebase.firestore.FieldValue.serverTimestamp;
+const projectFirestore = getFirestore();
+const timestamp = serverTimestamp();
```

Comment:
can we possibly deprecate this abastraction?

Reply:
It's possible, but I believe it's better to leave it that way. It's more simple to import the constant from the config file than explicitly import those functions from "firebase/firestore" on every file that need them. What do you think?

Reply:
LGTM. It's good to merge as is.